src/OSPSuite.Presentation/Presenters/SensitivityAnalyses/SensitivityAnalysisParameterSelectionPresenter.cs

```diff
@@ -123,11 +141,7 @@ public int IconFor(ISimulation simulation)
 
       public ISimulation Simulation => _sensitivityAnalysis.Simulation;
 
-      private static bool isNonFormulaParameter(ParameterSelection parameterSelection)
-      {
-         var parameter = parameterSelection.Parameter;
-         return parameter.IsFixedValue || !parameter.Formula.IsExplicit();
```

Comment:
See comment above. This was the same because we are filtering parameters but I think this is better like that